docs/manifests/kubent-cronjob.yaml

```
@@ -0,0 +1,64 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  namespace: kubent-system
+  name: kubent-reader
+rules:
+- apiGroups: [""]
+  resources: ["configmaps", "secrets"]
```

Just wanted to say that this is only enough for helm. Cluster collector silently ignores access denied and reports 0 resources discovered. 

```suggestion
- apiGroups: ["*"]
  resources: ["*"]
```